core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2468,4 +2468,10 @@ private CarbonCommonConstants() {
    * index server temp folder aging period default value 3hours.
    */
   public static final String CARBON_INDEXSERVER_TEMPFOLDER_DELETETIME_DEFAULT = "10800000";
+
+  public static final String STRING_LENGTH_EXCEEDED_MESSAGE =
+      "Record %s of column %s exceeded " + MAX_CHARS_PER_COLUMN_DEFAULT +
+          " characters. Please consider long string data type.";
+
+  public static final String FORCE_BAD_RECORD_ACTION = "FORCE";
```

Comment:
This string constant do not seem be used anywhere. Please remove it.

Reply:
done